src/Compilers/CSharp/Portable/CodeGen/EmitAddress.cs

```diff
@@ -103,7 +102,6 @@ private LocalDefinition EmitAddress(BoundExpression expression, AddressKind addr
         private void EmitComplexConditionalReceiverAddress(BoundComplexConditionalReceiver expression)
         {
             Debug.Assert(!expression.Type.IsReferenceType);
-            Debug.Assert(!expression.Type.IsValueType);
```

Comment:
Actually no-structs assert is still valid here since we do not need complex receiver for nullables. I will put the assert back.